templates/vue/src/store/mutations.js

```diff
@@ -37,12 +37,9 @@ export function addNode(state, node) {
 export function updateNode(state, payload) {
   const nodeIndex = Helpers.findNodeIndex(payload.id, state)
   const oldNode = state.nodes[nodeIndex]
```

Comment:
can we rename this to just `thisNode` or something since "old" doesn't make sense anymore?